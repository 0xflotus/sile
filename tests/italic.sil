\begin[papersize=a5]{document}
\font[style=italic]{Italic?}
\end{document}