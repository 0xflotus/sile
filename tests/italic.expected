<tests/italic.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.9764
My 	37.1857
Set font 	Gentium Plus;10;400;italic;normal;;LTR
T	44 87 68 79 76 70	(Italic)
Mx 	60.0389
T	34	(?)
[1] Mx 	207.4176
My 	553.7327
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20	(1)
End page
Finish

